### DISCUSSION TITLE
Fixed position of navigation

### DISCUSSION THREAD

**Discussion by maksymvasylyev** [neuefische/questions#623] [Web] [answered]
Hello
I made 3 grid rows.
By SE view Headline and Navigation are fixed.
But by other views (for Ex. XR-view) - Headline disappears when I reach the bottom of the page;
and navigation disappears when I reach the top of the page.
Also navigation disappears when I scroll fast.
Will be thankful for your advice. 

https://github.com/maksymvasylyev/capstone-project

Best Regards
Max

**Comment by onemanwenttomow** [comment, marked as answer]
Hi @maksymvasylyev 

I guess there are several ways we could attempt to tackle this. Here is one possible suggestion. 

Remove the following from the `div` that wraps the entire application. 

``` css
/* height: 190vh; 
display: grid; 
text-align: center; 
grid-template-rows: 3rem auto 3em;  */
```

remove the following from `main`

``` css
/* overflow-y: scroll; */
```

We can then make the `h1` and the `nav` fixed. 

`h1`
``` css
position: fixed;
top: 0; /* tell it where to position itself */
height: 3rem; /* need to set a height */
width: 100%; /* need to tell it to take up the full width of the page, otherwise it will only take up the amount of space it needs */
z-index: 10;
```

`nav`

``` css
position: fixed;
bottom: 0;
height: 3rem;
```

And finally we can add some breathing space above and below `main` (i.e. the height of your `h1` and `nav`.

`main`
``` css
padding-top: 3rem;
padding-bottom: 3rem;
```

This may then lead to styling issues that result in these changes.

**Reply by maksymvasylyev** [reply]
Thank you very much, we found solution, but it was not easy 👍🏼🦾
